src/components/menu/test/menu.spec.ts

```diff
@@ -116,6 +116,42 @@ describe('MenuController', () => {
       expect(menu).toEqual(someMenu);
     });
 
+    it('should get the only left menu when is not rtl', () => {
+      let someMenu = mockMenu();
+      someMenu.side = 'start';
+      menuCtrl._register(someMenu);
+
+      let menu = menuCtrl.get('left');
+      expect(menu).toEqual(someMenu);
+    });
+
+    it('should get the only right menu when is rtl', () => {
```

Comment:
I add the platform in mockMenu but I don't now how to access it. I think this
```
let someMenu = mockMenu();
      someMenu._plt.setDir('rtl', true);
```
but I don't know if it's right